src/components/forms/Checkbox/Checkbox.tsx

```diff
@@ -30,7 +34,13 @@ const { size } = shameStyles.checkbox
 /**
  * Use when the user needs to choose a single item in a list.
  */
-export const Checkbox: React.FC<CheckboxProps> = ({ label, checked, onChange, helpText }) => {
+export const Checkbox: React.FC<CheckboxProps> = ({
+  label,
+  checked,
+  helpText,
+  disabled = false,
```

Comment:
Small enough change, but going forward I think it would be better to have separate PRs to make sure they stay small and focused! :)

Reply:
Of course :slightly_smiling_face: